Assets/Scripts/MenuNavigation.cs

```diff
@@ -11,6 +11,7 @@ public void StartGame()
     {
         SceneManager.LoadScene(SceneManager.GetActiveScene().buildIndex+1);
         Debug.Log("Game Started");
+
```

Comment:
It is better to avoid making unnecessary changes to files, since they could result in merge conflicts. It is probably better to undo this change to prevent that

Reply:
Ok. How should I go about removing changes to these files?

Reply:
You have a multiple options. Here are some I can think of now:

1. Since this has the line number, one way is to just remove the exact line that was added in your editor locally (commit that locally and push to your remote branch and It'll show up here in this pull request)
2. Another (more involved) way to do this is to revert all changes in this file. Figure out which commit changed this file, grab the hash of the commit just before it, and use it with the following command: `git checkout <enter-commit-hash-here> -- file1/to/restore` 
3. You could probably just remove the whitespace directly via GitHub, but I don't recommend that in general because it is very prone to error (you'd never want to do that with code).

I'd use the first method for this case. The second is just in case you're curious about how to handle more complicated situations.